.github/workflows/ci.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
         rust: [stable, beta]
+    env:
+      CARGO_INCREMENTAL: 0
```

Comment:
Should we include this for all build steps, for consistency? @teor2345 thoughts?
Not a blocker to this PR though.

Reply:
Do you think we'll ever do anything that will break incremental builds?

Maybe we can disable incremental on "build" to speed up windows and macOS, but leave "build-chain-no-features" as incremental?

Reply:
This is now #1743